src/components/WebcamCapture/ViewFinderSVG.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SIZE = 768;
+const BORDER_WIDTH = 72;
+const HUD_INITIAL_LENGTH = 64;
+const HUD_STROKE_WIDTH = 8;
+
+const ViewFinder = props => {
+  const bw = BORDER_WIDTH;
```

Comment:
Can we not abbreviate these variables? I appreciate it adds more characters but is much easier to follow and avoids looking up what things are again when dealing with `hudStrokeWidth` instead of `hw`

Reply:
Yeah no problem, made working on the code really confusing with long names that's the only reason they're in there at all